app/assets/stylesheets/home.scss

```diff
@@ -4,15 +4,31 @@
 
 body {
   font-family: 'Helvetica';
-
+  max-width: 1200px;
+  margin: 0 auto;
   text-align: center;
 }
 
+header {
+  background: url('rubyrails') no-repeat center;
```

Comment:
Hey, I deployed this pull request but it doesn't look like the background is showing up.

https://railsn00bs-home-pr-9.herokuapp.com/

I think you need to change `url` to `image_url` :smile: